templates/zerver/help/user-groups.md

```diff
@@ -27,9 +27,22 @@ trying to send a message to a group of people, you'll want to either
 1. Add or remove users (including yourself). Click outside the box
    to save.  Zulip will notify everyone who is added or removed.
 
+{end_tabs}
+
+### Delete a user group
+
+{start_tabs}
+
+{settings_tab|user-groups-admin}
+
+1. Find the group.
+
+1. Click the trash icon (<i class="fa fa-trash-o"></i>) in the lower right corner of the group section.
```

Comment:
A few corrections are needed to make it consistent with our style guidelines and other docs:
```suggestion
1. Click on the trash (<i class="fa fa-trash-o"></i>) icon in the top right corner of the user group.
```